### ISSUE TITLE
IndexGenome issue

### ISSUE THREAD

**Issue by sudeepsahadevan** [erhard-lab/gedi#15]
Hi Florian,
I was not able to reproduce the genome indexing steps from wiki. What I noticed when I supply gtf file or use `-genome` and `-version` options  `IndexGenome`  goes to sleep or disk read status after writing `<gtf>.genes.tab` and `<gtf>.transcripts.tab` files. If it helps, at this step Price also generated 0 size files with the naming convention `<gtf>.index.cit.1+.CITI7<some_number>.data`. But, if I do not supply the annotation file and use `IndexGenome -s <fasta>`  Price starts generating indices. Could you please look into it ?

Thank you in advance,

**Comment by florianerhard** [comment]
Hi,
could you give me more information on what exactly you tried? What was the gtf file? Which organism did you try for `-organism` (not `-genome` I suppose?). Which version of Gedi did you use, and which operating system? It would be great to have a minimal example where this fails...

Best,
Florian

**Comment by sudeepsahadevan** [comment]
Hi Florian,

I was in fact trying to replicate **Prepare a genome** section from Price wiki. So first what I used was this:
`gedi -e IndexGenome -organism homo_sapiens -version 75` 
The steps ran until the files `homo_sapiens.75.genes.tab` and `homo_sapiens.75.transcripts.tab` were generated. After generating the cit file (file size: 0) with format `homo_sapiens.75.index.cit.1+.CITI<someNumber>.data` the process seems to be in sleep/disk read state and did nothing even when left to run for a couple of hours. So, I began poking around and what I noticed was that gedi/Price behaves similarly (creates .tab files and CIT file, then process in sleep or disk read) for `gedi -e IndexGenome -s homo_sapiens.75.fasta -a homo_sapiens.75.gtf` where the fasta and gtf files were downloaded in step above (ie`gedi -e IndexGenome -organism homo_sapiens -version 75` ). But, I also noticed here that if I remove the gtf file, ie `gedi -e IndexGenome -s homo_sapiens.75.fasta`  gedi appears to run and create indices. Gedi version is 1.0.1 and java is 
`java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)`

I hope it is more clear now.

Thank you,
Sudeep.
